core/prelude-editor.el

```diff
@@ -358,14 +358,20 @@ indent yanked text (with prefix arg don't indent)."
 ;; enable Prelude's keybindings
 (prelude-mode t)
 
-;; supercharge your undo/redo with undo-tree
-(require 'undo-tree)
-;; autosave the undo-tree history
-(setq undo-tree-history-directory-alist
-      `((".*" . ,temporary-file-directory)))
-(setq undo-tree-auto-save-history t)
-(global-undo-tree-mode)
-(diminish 'undo-tree-mode)
+(defun prelude-enable-undo-tree ()
```

Comment:
I guess `maybe-enable` would be a better name here to indicate that the enabling is conditional.

Reply:
I was following the name pattern of other functions in this module, like `prelude-enable-whitespace`, but I agree, I think maybe-enable is more coherent.

Reply:
Done